rxbinding-support-v4/src/androidTest/java/com/jakewharton/rxbinding2/support/v4/view/RxMenuItemCompatTest.java

```diff
@@ -33,7 +33,7 @@
 
   @Test @UiThreadTest public void actionViewEvents() {
     RecordingObserver<MenuItemActionViewEvent> o = new RecordingObserver<>();
-    Subscription subscription = RxMenuItemCompat.actionViewEvents(menuItem).subscribe(o);
+    Disposable subscription = RxMenuItemCompat.actionViewEvents(menuItem).subscribeWith(o);
```

Comment:
I've added `Disposable` interface to `RecordingObserver` so I can call `dispose` directly on it. This way I also subscribe with simple `subscribe` method instead of `subscribeWith`. Ex 9f7f0a309f6934cf9e14a8a3044b072e0b7062e5
Do you think this matters? Should I change it?